apis/ec2/src/main/java/org/jclouds/ec2/compute/domain/EC2HardwareBuilder.java

```diff
@@ -429,6 +444,89 @@ public static EC2HardwareBuilder t2_large() {
             .processors(ImmutableList.of(new Processor(1.0, 0.4))).rootDeviceType(RootDeviceType.EBS);
    }
 
+   /**
+    * @see InstanceType#T2_XLARGE
+    */
+   public static EC2HardwareBuilder t2_xlarge() {
+      return new EC2HardwareBuilder(InstanceType.T2_XLARGE).t2()
+              .ram(16384)
+              .processors(ImmutableList.of(new Processor(2.0, 0.4))).rootDeviceType(RootDeviceType.EBS);
+   }
+
+   /**
+    * @see InstanceType#T2_2XLARGE
+    */
+   public static EC2HardwareBuilder t2_2xlarge() {
+      return new EC2HardwareBuilder(InstanceType.T2_2XLARGE).t2()
+              .ram(32768)
+              .processors(ImmutableList.of(new Processor(4.0, 0.4))).rootDeviceType(RootDeviceType.EBS);
+   }
+
+   /**
+    * @see InstanceType#T3_NANO
+    */
+   public static EC2HardwareBuilder t3_nano() {
+      return new EC2HardwareBuilder(InstanceType.T3_NANO).t3()
+              .ram(512)
+              .processors(ImmutableList.of(new Processor(1.0, 0.4))).rootDeviceType(RootDeviceType.EBS);
+   }
+
+   /**
+    * @see InstanceType#T3_MICRO
+    */
+   public static EC2HardwareBuilder t3_micro() {
+      return new EC2HardwareBuilder(InstanceType.T3_MICRO).t3()
+              .ram(1024)
+              .processors(ImmutableList.of(new Processor(1.0, 0.4))).rootDeviceType(RootDeviceType.EBS);
+   }
+
+   /**
+    * @see InstanceType#T3_SMALL
+    */
+   public static EC2HardwareBuilder t3_small() {
+      return new EC2HardwareBuilder(InstanceType.T3_SMALL).t3()
+              .ram(2048)
+              .processors(ImmutableList.of(new Processor(1.0, 0.4))).rootDeviceType(RootDeviceType.EBS);
+   }
+
+   /**
+    * @see InstanceType#T3_MEDIUM
+    */
+   public static EC2HardwareBuilder t3_medium() {
+      return new EC2HardwareBuilder(InstanceType.T3_MEDIUM).t3()
+              .ram(4096)
+              .processors(ImmutableList.of(new Processor(1.0, 0.4))).rootDeviceType(RootDeviceType.EBS);
+   }
+
+   /**
+    * @see InstanceType#T3_LARGE
+    */
+   public static EC2HardwareBuilder t3_large() {
+      return new EC2HardwareBuilder(InstanceType.T3_LARGE).t3()
+              .ram(8192)
+              .processors(ImmutableList.of(new Processor(1.0, 0.4))).rootDeviceType(RootDeviceType.EBS);
```

Comment:
T3 instances has number of vCPU is wrong https://aws.amazon.com/it/ec2/instance-types/t3/
The javadoc is ok at ```InstanceType```, though

Could you please review them?